setup/package.sh

```diff
@@ -47,64 +101,105 @@ prepare () {
   cp default.xml out/conf
   cp traccar.xml out/conf
 
-  innoextract innosetup-*.exe
-  echo "If you got any errors here try isetup version 5.5.5 (or check supported versions using 'innoextract -v')"
+  if [ $PLATFORM = "all" -o $PLATFORM = "windows" ]; then
+	innoextract innosetup-*.exe >/dev/null
+	info "If you got any errors here try Inno Setup version 5.5.5 (or check supported versions using 'innoextract -v')"
+  fi
 }
 
 cleanup () {
+  info "Cleanup"
   rm ../traccar-web/web/app.min.js
 
   rm -r out
-  rm -r tmp
-  rm -r app
+  if [ $PLATFORM = "all" -o $PLATFORM = "windows" ]; then
+	rm -r tmp
+	rm -r app
+  fi
 }
 
 package_other () {
+  info "Building Zip archive"
   cp README.txt out
   cd out
-  zip -r ../traccar-other-$VERSION.zip *
+  zip -q -r ../traccar-other-$VERSION.zip *
   cd ..
   rm out/README.txt
+  ok "Created Zip archive"
 }
 
 package_windows () {
-  unzip -o java-*.windows.x86_64.zip
+  info "Building Windows 64 installer"
+  unzip -q -o java-*.windows.x86_64.zip
   jlink --module-path java-*.windows.x86_64/jmods --add-modules java.se.ee --output out/jre
   rm -rf java-*.windows.x86_64
-  wine app/ISCC.exe traccar.iss
+  wine app/ISCC.exe traccar.iss >/dev/null
   rm -rf out/jre
-  zip -j traccar-windows-64-$VERSION.zip Output/traccar-setup.exe README.txt
+  zip -q -j traccar-windows-64-$VERSION.zip Output/traccar-setup.exe README.txt
   rm -r Output
+  ok "Created Windows 64 installer"
 }
 
-package_unix () {
+package_linux () {
   cp setup.sh out
   cp traccar.service out
 
-  unzip -o jdk-*-linux-x64.zip
-  jlink --module-path jdk-*-linux-x64/jmods --add-modules java.se.ee --output out/jre
-  rm -rf jdk-*-linux-x64
-  makeself --notemp out traccar.run "traccar" ./setup.sh
+  unzip -q -o jdk-*-linux-$1.zip
+  jlink --module-path jdk-*-linux-$1/jmods --add-modules java.se.ee --output out/jre
+  rm -rf jdk-*-linux-$1
+  makeself --quiet --notemp out traccar.run "traccar" ./setup.sh
   rm -rf out/jre
-  zip -j traccar-linux-64-$VERSION.zip traccar.run README.txt
-  rm traccar.run
+}
 
-  unzip -o jdk-*-linux-armhf.zip
-  jlink --module-path jdk-*-linux-armhf/jmods --add-modules java.se.ee --output out/jre
-  rm -rf jdk-*-linux-armhf
-  makeself --notemp out traccar.run "traccar" ./setup.sh
-  rm -rf out/jre
-  zip -j traccar-linux-arm-$VERSION.zip traccar.run README.txt
+cleanup_linux () {
   rm traccar.run
-
   rm out/setup.sh
   rm out/traccar.service
 }
 
+package_linux_x64 () {
+  info "Building Linux 64 installer"
+  package_linux x64
+  zip -q -j traccar-linux-64-$VERSION.zip traccar.run README.txt
```

Comment:
And this as well.

Reply:
I think you misunderstood what I meant. You can have that like in the `package_linux` function, but you will still build per platform like you do now.